xml/Menu/civiawards.xml

```diff
@@ -3,9 +3,16 @@
   <item>
     <path>civicrm/awardreview</path>
     <page_callback>CRM_CiviAwards_Form_AwardReview</page_callback>
-    <title>AwardReview</title>
+    <title>Award Review</title>
     <access_arguments>access CiviCRM</access_arguments>
   </item>
+  <item>
+    <path>civicrm/award-review-ssp</path>
+    <page_callback>CRM_CiviAwards_Form_AwardReview</page_callback>
+    <title>Award Review</title>
+    <access_arguments>access applicant portal</access_arguments>
```

Comment:
@tunbola we are using "CiviAwards: Access awards panel portal" in SSP Drupal for Applications review listing and individual pages so I think we should use same here ?